### DISCUSSION TITLE
"Server Error" on uploading the photo on localhost

### DISCUSSION THREAD

**Discussion by 87prashant** [ParabolInc/parabol#6848] [Q&A] [answered]
### Any help?
![Screenshot 2022-07-04 185527](https://user-images.githubusercontent.com/106697681/177164633-0e8bd90f-3b43-4d79-b9b8-fb8aa8e7c5a1.png)

**Comment by tianrunhe** [comment, marked as answer]
Hi @87prashant, please add/modify this line in your `.env` (unless you'd like to test S3 storage):

```
FILE_STORE_PROVIDER='local'
```

**Reply by 87prashant** [reply]
Thanks for helping @tianrunhe 
Oh! I haven't visited [this](https://github.com/ParabolInc/parabol/blob/master/docker/README.md)
